test/kino_db/connection_cell_test.exs

```diff
@@ -214,6 +260,11 @@ defmodule KinoDB.ConnectionCellTest do
 
     assert_smart_cell_update(kino, %{"type" => "mysql", "port" => 3306}, """
     opts = [hostname: "localhost", port: 3306, username: "", password: "", database: ""]
+
+    if opts[:ssl] == true do
+      :ssl.start()
+    end
```

Comment:
Instead of having a conditional, we should either add startup or not when we generate the code.

Also probably better to call `Application.ensure_all_started(:ssl)` :)